builtin/files/cluster.yaml.tmpl

```diff
@@ -1179,13 +1179,25 @@ kubernetes:
     enabled: false
 
 #  controllerManager:
+#   resources:
+#     requests:
+#       cpu: 100m
+#       memory: 100M
+#     limits:
+#       cpu: 250m
+#       memory: 512M
+
+#  apiServer:
+#    # Memory limit for apiserver in MB (used to configure sizes of etcd caches, etc.)
+#    targetRamMb: 320M
+#    # Resource limits for the apiserver.
 #    resources:
 #      requests:
-#        cpu: 100m
-#        memory: 100M
+#        cpu: 256M
+#        memory: 256M
 #      limits:
-#        cpu: 250m
-#        memory: 512M
+#        cpu: 800M
+#        memory: 2000M
```

Comment:
Please can you use Mi for memory and m (mili) for cpu?

Reply:
Updated.